### PR TITLE
Work around Amazon Linux 2 repository issue

### LINKED ISSUE #175: There is no SIMP repo for Amazon Linux 2
Need a workaround for the fact that there is currently no Amazon Linux 2 repository.

### PR DESCRIPTION
**Pull request by trevor-vaughan**
Closes #175

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # ------------------------------------------------------------------------------
 # The testing matrix considers ruby/puppet versions supported by SIMP and PE:
+      if (( os == 'Amazon' ) && ( "#{release}" == '2' ))
```
Comment (op-ct) [inline .gitlab-ci.yml:3]: This and breaks the `.gitlab-ci.yml` and looks like it's probably misplaced Ruby copypasta meant for something else.

```suggestion
```

**Review by op-ct — CHANGES_REQUESTED** [review]
This still fails on Amazon Linux 2: https://gitlab.com/simp/rubygem-simp-beaker-helpers/-/jobs/2260389857 ― the URLs in the `.repo` files delivered by simp-release-community use `$releasever`, which still resolves to `2` despite the workaround in `install_simp_repos`.  

I'm taking a look adding a fix to the upstream repo for simp-release-community.

**Comment by op-ct** [comment]
Submitted an upstream MR to the simp-release-* RPMs

**Comment by trevor-vaughan** [comment]
@op-ct I had some bad logic in the workaround code. Just rebased and submitted a fix that appears to work.

**Review by op-ct — APPROVED** [review]
The hack-around is sketch, but it gets us far enough to get to the next steps.

**Pull request merged**